mkdocs.yml

```
@@ -3,19 +3,38 @@ site_name: OODEEL
 # Set navigation here
 nav:
   - Home: index.md
+  - Tutorials:
+    - "Getting started: Maximum Logit Score on MNIST": pages/demo_experiment.ipynb
+    - Important Note On Feature Extractors: pages/feature_extractor_tuto.ipynb
+    - OOD Baselines (Tensorflow):
+      - ODIN: notebooks/tensorflow/demo_odin.ipynb
+      - DKNN: notebooks/tensorflow/demo_dknn.ipynb
+      - VIM: notebooks/tensorflow/demo_vim.ipynb
+      - Energy: notebooks/tensorflow/demo_energy.ipynb
```

The notebooks are called `demo_[]_tf.py`.
Moreover, there are two more notebooks in the `torch` and `tensorflow` folders: Mahalanobis and MLS/MSP. We can maybe add them to the navigation bar?
```suggestion
      - ODIN: notebooks/tensorflow/demo_odin_tf.ipynb
      - DKNN: notebooks/tensorflow/demo_dknn_tf.ipynb
      - VIM: notebooks/tensorflow/demo_vim_tf.ipynb
      - Energy: notebooks/tensorflow/demo_energy_tf.ipynb
```